lib/dry/types/hash/schema.rb

```diff
@@ -149,8 +149,6 @@ def resolve(hash)
           end
         end
       end
-
-      private_constant(*constants(false))
```

Comment:
this was a quick and dirty hack to see if the specs will pass on rubinius. Regardless of this, what's the point of making constants private? Before this I didn't even know it's possible :) Shouldn't we just document them as private?